build-script-macos-02.sh

```diff
@@ -200,7 +200,7 @@ build_06_build_dependency_qt() {
       -skip qtscript -skip qtscxml -skip qtsensors -skip qtserialbus -skip qtspeech \
       -skip qttranslations -skip qtwayland -skip qtwebchannel -skip qtwebengine -skip qtwebglplugin \
       -skip qtwebsockets -skip qtwebview -skip qtwinextras -skip qtx11extras -skip qtxmlpatterns \
-      QMAKE_APPLE_DEVICE_ARCHS=`arch`
+      QMAKE_APPLE_DEVICE_ARCHS="$(uname -m)"
```

Comment:
Don't modify the shell scripts.  They are automatically rebuilt using the `create_scripts.yml` workflow.

Reply:
Modified `build_deps.yml` (and found an extra `=` in the `CXX` assignment)